### PR TITLE
Company Cards , Each Company Section Completed 

### PR DESCRIPTION
**Pull request by Pradeep-Kumar-Rebbavarapu**
few small bugs need to be fixed

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+GOOGLE_ID = "786625929891-uoi3ro7vi4fh7m3hprrkptg1tt9kkggv.apps.googleusercontent.com"
```
Comment (akashg3627) [inline .env:1]: never commit these

Comment (Pradeep-Kumar-Rebbavarapu) [reply .env:1]: Ok bhaiya

File: src/app/(routes)/admin/[adminId]/layout.tsx
Comment (harshtalati2410) [inline src/app/(routes)/admin/[adminId]/layout.tsx:1]: remove the comments on top. Not needed.

File: src/app/(routes)/admin/company/[companyId]/page.tsx
Comment (harshtalati2410) [inline src/app/(routes)/admin/company/[companyId]/page.tsx:1]: Two changes:
1. Use class components format for EachCompanyPage component
2. fetchEachCompanyDetails function should be kept in the helpers folder and imported from there.

File: src/app/(routes)/admin/company/page.tsx
Comment (harshtalati2410) [inline src/app/(routes)/admin/company/page.tsx:1]: Keep fetchCompanyFromTheSeason function into the helpers folder and import when needed.

File: src/app/(routes)/admin/layout.tsx
Comment (harshtalati2410) [inline src/app/(routes)/admin/layout.tsx:1]: Remove unnecessary comments from the top.

File: src/components/Students/StudentsTable/StudentTable.tsx
Comment (harshtalati2410) [inline src/components/Students/StudentsTable/StudentTable.tsx:1]: Use class components to declare StudentTable component.

File: src/components/company/EachCompanyCard.tsx
Comment (harshtalati2410) [inline src/components/company/EachCompanyCard.tsx:1]: Use class component to declare the component
Change the name ele used in the interface companyItem.

File: src/components/company/JAFCard.tsx
Comment (harshtalati2410) [inline src/components/company/JAFCard.tsx:1]: Use class component to declare the component

File: src/app/(routes)/admin/Jobs/error.tsx
[the remark this answers is not in the document]
Comment (Pradeep-Kumar-Rebbavarapu) [reply src/app/(routes)/admin/Jobs/error.tsx:1]: done

File: src/app/(routes)/admin/[adminId]/layout.tsx
Comment (Pradeep-Kumar-Rebbavarapu) [reply src/app/(routes)/admin/[adminId]/layout.tsx:1]: done

File: src/app/(routes)/admin/company/page.tsx
Comment (Pradeep-Kumar-Rebbavarapu) [reply src/app/(routes)/admin/company/page.tsx:1]: done

File: src/app/(routes)/admin/layout.tsx
Comment (Pradeep-Kumar-Rebbavarapu) [reply src/app/(routes)/admin/layout.tsx:1]: done

File: src/components/SideBar/DropDowns/SessionDropDown.tsx
[the remark this answers is not in the document]
Comment (Pradeep-Kumar-Rebbavarapu) [reply src/components/SideBar/DropDowns/SessionDropDown.tsx:1]: done

File: src/components/company/CompanyCard.tsx
[the remark this answers is not in the document]
Comment (Pradeep-Kumar-Rebbavarapu) [reply src/components/company/CompanyCard.tsx:1]: done

File: src/components/company/EachCompanyCard.tsx
Comment (Pradeep-Kumar-Rebbavarapu) [reply src/components/company/EachCompanyCard.tsx:1]: done

File: src/app/(routes)/admin/company/[companyId]/page.tsx
Comment (Pradeep-Kumar-Rebbavarapu) [reply src/app/(routes)/admin/company/[companyId]/page.tsx:1]: done

File: src/components/jobs/JobCard.tsx
[the remark this answers is not in the document]
Comment (Pradeep-Kumar-Rebbavarapu) [reply src/components/jobs/JobCard.tsx:1]: done

**Review by harshtalati2410 — APPROVED** [review]

**Pull request merged**